lib/web/integrations_awsoidc.go

```diff
@@ -109,10 +109,14 @@ func (h *Handler) awsOIDCDeployService(w http.ResponseWriter, r *http.Request, p
 		return nil, trace.Wrap(err)
 	}
 
-	databaseAgentMatcherLabels := make(types.Labels, len(req.DatabaseAgentMatcherLabels))
+	databaseAgentMatcherLabels := make(types.Labels, len(req.DatabaseAgentMatcherLabels)+3)
 	for _, label := range req.DatabaseAgentMatcherLabels {
 		databaseAgentMatcherLabels[label.Name] = utils.Strings{label.Value}
 	}
+	// Add additional default labels.
+	databaseAgentMatcherLabels[types.DiscoveryLabelVPCID] = []string{req.VPCID}
+	databaseAgentMatcherLabels[types.DiscoveryLabelRegion] = []string{req.Region}
+	databaseAgentMatcherLabels[types.DiscoveryLabelAccountID] = []string{req.AccountID}
```

Comment:
We are not checking this variables.
Does the UI always send them?

Reply:
thanks for catching this. here is the [change](https://github.com/gravitational/teleport/pull/45180/commits/678f00b374a72a62bc37061b709a3b2086e40ee0)

i left the merging of labels as is, just in case we want to bring back custom labels, but atm the UI will no longer allow any custom labels